browsers/chrome.json

```diff
@@ -824,6 +824,11 @@
           "status": "nightly",
           "engine": "Blink",
           "engine_version": "118"
+        },
+        "119": {
+          "status": "planned",
```

Comment:
This is actually the nightly build but MDN only seems to have stable, beta and nightly with no dev channel option so leaving as planned for now.